tools/tertiary-analysis/sc3/sc3-sc3-calc-transfs.xml

```diff
@@ -1,39 +1,34 @@
-<tool id="sc3-sc3-calc-transfs" name="SC3 calculate transformations" version="0.1.0">
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="sc3-sc3-calc-transfs" name="SC3 calculate transformations" version="@TOOL_VERSION@+galaxy0">
     <description>Calculate CPM from serialized scater R object</description>
     <macros>
-        <import>macros.xml</import>
+        <import>sc3-macros.xml</import>
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
-        sc3-sc3-calc-transfs.R -i R_scater_serialized.rds -o R_scater_cpm.rds
+        sc3-sc3-calc-transfs.R -i R_scater_serialized.rdata -o R_scater_cpm.rdata
```

Comment:
Does this command work? Perhaps link "input1" to `R_scater_serialized.rdata` first?

Reply:
It is working now. Thanks for the suggestion.